Servers/UI/OJS.Servers.Ui/ClientApp/src/components/administration/checkers/checker-form/CheckerForm.tsx

```diff
@@ -56,6 +59,14 @@ const CheckerForm = (props: ICheckerFormProps) => {
         },
     ] = useUpdateCheckerMutation();
 
+    useEffect(() => {
+        if (isSuccessfullyCreated && onSuccess) {
+            setTimeout(() => {
+                onSuccess();
+            }, 500);
+        }
+    }, [ isSuccessfullyCreated, onSuccess ]);
```

Comment:
This whole useEffect is repeated in many components. Can you extract it in a custom hook?
```
useDelayedSuccessEffect({
    isSuccessfullyCreated,
    onSuccess,
    delay: 500, // Optional (with a default value)
  });
```

And don't forget to clear the timout interval in the hook with
`return () => clearTimeout(timeoutId);`

Reply:
Done.